GVFS/GVFS/CommandLine/MountVerb.cs

```diff
@@ -114,35 +114,27 @@ protected override void Execute(GVFSEnlistment enlistment)
 
                 if (!GVFSPlatform.Instance.KernelDriver.IsReady(tracer, enlistment.EnlistmentRoot, this.Output, out errorMessage))
                 {
-                    if (GVFSPlatform.Instance.UnderConstruction.SupportsGVFSService)
-                    {
-                        tracer.RelatedEvent(
-                            EventLevel.Informational,
-                            $"{nameof(MountVerb)}_{nameof(this.Execute)}_EnablingKernelDriverViaService",
-                            new EventMetadata
-                            {
-                                { "KernelDriver.IsReady_Error", errorMessage },
-                                { TracingConstants.MessageKey.InfoMessage, "Service will retry" }
-                            });
+                    tracer.RelatedEvent(
+                        EventLevel.Informational,
+                        $"{nameof(MountVerb)}_{nameof(this.Execute)}_EnablingKernelDriverViaService",
+                        new EventMetadata
+                        {
+                            { "KernelDriver.IsReady_Error", errorMessage },
+                            { TracingConstants.MessageKey.InfoMessage, "Service will retry" }
+                        });
 
+                    if (GVFSPlatform.Instance.ServiceSupportsDriverLoading())
```

Comment:
I noticed that this is the only spot that we're calling `ServiceSupportsDriverLoading`.

In the interests of keeping the code as similar as possible on both platforms, would it be cleaner to send the "attach" message on Mac as well and have it be a no-op in the service?

If we do decide to leave this check in, does it also need to be added to `ServiceVerb.Execute` (which also calls `TryEnableAndAttachPrjFltThroughService`)?

/ cc: @nickgra and @kewillford for their thoughts as well, this is potentially related to the discussion on renaming `TryEnableAndAttachPrjFltThroughService` below.

Reply:
I would prefer sending the attach message as well on macOS and simply having the service 'no-op' it rather than putting all of this code into the MountVerb itself. I think it would make the amount of platform specific code we have cleaner and that has been a goal of mine when working on cross platform code.

Reply:
Updated `GVFSService.Mac.HandleMessage` to do a NOOP for `EnableAndAttachProjFSRequest` messages.

Also removed `GVFSPlatform. ServiceSupportsDriverLoading`